src/handlebar-helpers.js

```diff
@@ -23,6 +23,14 @@ const helpers = {
   },
   or() {
     return Array.prototype.slice.call(arguments, 0, -1).some(Boolean);
+  },
+  split: function (data, delimiter) {
+    if (typeof data === 'string' && typeof delimiter === 'string') {
+      if (delimiter !== '') {
+        return data.split(delimiter);
+      }
+    }
+    return [];
```

Comment:
instead of empty array, would it make more sense to return the original data, but not split at all?

Reply:
Done